server/vue-middleware.js

```diff
@@ -74,7 +74,8 @@ module.exports = function createMiddleware({
 		// set html response headers
 		res.setHeader('Content-Type', 'text/html');
 		// Set strict cache-control headers for protected pages
-		if (req?.url?.match(/(settings|portfolio|lend\/saved-search)/g)?.length) {
+		// eslint-disable-next-line max-len
+		if (req?.url?.match(/(checkout|settings|portfolio|lend\/saved-search|monthlygood\/thanks|process-browser-auth|register|start-verification|confirm-instant-donation|instant-donation-thanks)/g)?.length) {
```

Comment:
To be clear, I do not like this. Prob move it to a simple list util soon or find a better way to import/clean our route declarations...